src/main/scala/it/pagopa/interop/attributeregistryprocess/api/impl/AttributeRegistryApiServiceImpl.scala

```diff
@@ -0,0 +1,177 @@
+    } yield result
+
+    onComplete(result) {
+      getAttributeByNameResponse[Attribute](operationLabel) { res =>
+        logger.info(s"Retrieved attribute with id${res.id}")
+        getAttributeByName200(res)
+      }
+    }
+  }
+
+  override def getAttributeByOriginAndCode(origin: String, code: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerAttribute: ToEntityMarshaller[Attribute],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE, API_ROLE) {
+    val operationLabel: String = s"Retrieving attribute $origin/$code"
+    logger.info(operationLabel)
+
+    val result: Future[Attribute] = for {
+      result <- attributeRegistryManagementService.getAttributeByOriginAndCode(origin, code).map(_.toApi)
+    } yield result
+
+    onComplete(result) {
+      getAttributeByOriginAndCodeResponse[Attribute](operationLabel) { res =>
+        logger.info(s"Retrieved attribute with id${res.id}")
+        getAttributeByOriginAndCode200(res)
+      }
+    }
+  }
+
+  override def getAttributes(search: Option[String])(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerAttributesResponse: ToEntityMarshaller[AttributesResponse],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE, API_ROLE) {
+    val operationLabel: String = s"Retrieving attributes by search string ${search.getOrElse("")}"
+    logger.info(operationLabel)
+
+    val result: Future[AttributesResponse] = for {
+      result <- attributeRegistryManagementService.getAttributes(search).map(_.attributes.map(_.toApi))
+    } yield AttributesResponse(attributes = result)
+
+    onComplete(result) {
+      getAttributesResponse[AttributesResponse](operationLabel)(getAttributes200)
+    }
+  }
+
+  override def getBulkedAttributes(ids: Option[String])(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerAttributesResponse: ToEntityMarshaller[AttributesResponse]
+  ): Route = authorize(ADMIN_ROLE, API_ROLE) {
+    val operationLabel: String = s"Retrieving attributes in bulk by identifiers in (${ids.getOrElse("")})"
+    logger.info(operationLabel)
+
+    val result: Future[AttributesResponse] = for {
+      result <- attributeRegistryManagementService.getBulkedAttributes(ids).map(_.attributes.map(_.toApi))
+    } yield AttributesResponse(attributes = result)
+
+    onComplete(result) {
+      getBulkedAttributesResponse[AttributesResponse](operationLabel)(getBulkedAttributes200)
+    }
+  }
+
+  override def loadCertifiedAttributes()(implicit contexts: Seq[(String, String)]): Route =
+    authorize(ADMIN_ROLE, API_ROLE) {
+      val operationLabel: String = s"Loading certified attributes from Party Registry"
+      logger.info(operationLabel)
+
+      val result: Future[Unit] = for {
+        result <- attributeRegistryManagementService.loadCertifiedAttributes()
```

Comment:
This should be implemented directly in this process and removed from management.

Reply:
You can implement it when you start the PIN-2963 task.